jest.config.js

```diff
@@ -0,0 +1,15 @@
+const config = {
+  collectCoverage: true,
+  collectCoverageFrom: ['lib/**/*.js'],
+  moduleDirectories: ['node_modules'],
+  setupFiles: ['./testSetup.js'],
+  snapshotSerializers: ['jest-snapshot-serializer-ansi'],
+  testEnvironment: 'node',
+  transform: {
+    '\\.[jt]sx?$': 'babel-jest',
+  },
+  /** Also transform ESM packages in `node_modules` */
+  transformIgnorePatterns: [],
```

Comment:
Jest doesn't support ESM modules yet (at least for mocking), so convert everything back to CommonJS for tests.